Iterator and generator/generator.py

```diff
@@ -0,0 +1,16 @@
+import nltk
+from nltk.corpus import words
+import random
+
+# for i in range(10):
+#     print(random.randrange(0,100, 1))
+
+def my_generator(count_words: int):
```

Comment:
![image](https://github.com/Gleb1395/Practice/assets/17034253/de7ca951-0a52-452d-9931-d2ebdf22841f)

Reply:
words are not uniq

Reply:
![image](https://github.com/Gleb1395/Practice/assets/17034253/91e85ffe-a05f-4754-b919-37399e29afb5)

Still not uniq for 10k words